src/appmixer/notion/core/NewDatabaseItem/component.json

```diff
@@ -0,0 +1,378 @@
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                            "Tags": {
+                                "label": "Tags",
+                                "value": "Tags",
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "id": {
+                                            "label": "Property ID",
+                                            "value": "id"
+                                        },
+                                        "type": {
+                                            "label": "Property Type",
+                                            "value": "type"
+                                        },
+                                        "multi_select": {
+                                            "label": "multi_select",
+                                            "value": "multi_select",
+                                            "schema": {
+                                                "type": "array",
+                                                "items": {
+                                                    "type": "object",
+                                                    "properties": {}
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                            "Created by": {
+                                "label": "Created by",
+                                "value": "Created by",
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "id": {
+                                            "label": "Property ID",
+                                            "value": "id"
+                                        },
+                                        "type": {
+                                            "label": "Property Type",
+                                            "value": "type"
+                                        },
+                                        "created_by": {
+                                            "label": "Created By",
+                                            "value": "created_by",
+                                            "schema": {
+                                                "type": "object",
+                                                "properties": {
+                                                    "object": {
+                                                        "label": "Creator Object",
```

Comment:
inside of the schema there should be a title (not label) 
see more here https://github.com/clientIO/appmixer-fe/issues/3861#issuecomment-1920015785